docs/quickstart/generate-report-ui.mdx

```diff
@@ -72,7 +72,7 @@ elementary:
       keyfile: [full path to your keyfile]
 
       project: [project id]
-      dataset: [dataset name]
+      dataset: [dataset name]_elementary
```

Comment:
I agree that we need to make it more clear that this is the elementary dataset.
The issue is that we don't expect "[dataset name]_elementary", this is just a suggestion, you can change it if you want. It will even work if you don't configure a custom schema. 
We are [adding this macro](https://github.com/elementary-data/dbt-data-reliability/commit/55e68ef2f26e2eabf7a9a0f176df0b3456536b12) to the next release, to make it easier.

What I suggest is that you change:
`dataset: [dataset name]_elementary`
to:
`dataset: [dataset name] # elementary dataset, usually [dataset name]_elementary`

wdyt?

Reply:
@Maayan-s I'm delighted to be the first to exercise your new docs workflow!

Thanks for the suggestion. I've applied it to all three databases for consistency in experience.